### DISCUSSION TITLE
Welcome to 1999AZZAR Discussions!

### DISCUSSION THREAD

**Discussion by 1999AZZAR** [1999AZZAR/1999AZZAR#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dirga810** [comment]
Google-Dorks-List-New-2020.txt (view on GitHub)

**Comment by Dirga810** [comment]
> 
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with other community members.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
> 

My name YD tolong bantu saya untuk masuk kedalam email istri saya, agar saya dpt membuktikan bahawa dia telash selingkuh dgn laki laki lain.thx

**Reply by Dirga810** [reply]
Help me

**Reply by 1999AZZAR** [reply]
Waduh sorry bang, kalo masalah itu bukan ranah saya
